app/src/main/java/com/zeapo/pwdstore/UserPreference.kt

```diff
@@ -374,6 +376,22 @@ class UserPreference : AppCompatActivity() {
             }
         }
 
+        private fun updateClearSavedPassphrasePrefs() {
+            clearSavedPassPreference?.apply {
+                val sshPass = encryptedPreferences.getString("ssh_key_local_passphrase", null)
+                val httpsPass = encryptedPreferences.getString("https_password", null)
+                if (sshPass == null && httpsPass == null) {
+                    isVisible = false
+                    return
+                }
+                title = when {
+                    sshPass != null -> getString(R.string.clear_saved_passphrase_ssh)
+                    httpsPass != null -> getString(R.string.clear_saved_passphrase_https)
```

Comment:
Interchange the two branches here, I think it happens far more frequently that somebody wants to clear their HTTPS passphrase than that of their SSH key.

Reply:
Fixed.